openlibrary/plugins/openlibrary/js/ile/utils/SelectionManager/SelectionManager.js

```diff
@@ -0,0 +1,273 @@
+// @ts-check
+import $ from 'jquery';
+import { move_to_work, move_to_author } from '../ol.js';
+import './SelectionManager.less';
+
+/**
+ * The SelectionManager is responsible for making things (e.g. books in search results,
+ * or authors on books pages) on Open Library selectable and drag/droppable from one
+ * Open Library window to another.
+ */
+export default class SelectionManager {
+    /**
+     * @param {import('../../index.js').IntegratedLibrarianEnvironment} ile
+     */
+    constructor(ile, curpath=location.pathname) {
+        this.ile = ile;
+        this.curpath = curpath;
+        this.inited = false;
+
+        this.toggleSelected = this.toggleSelected.bind(this);
+        this.dragStart = this.dragStart.bind(this);
+        this.onDrop = this.onDrop.bind(this);
+        this.allowDrop = this.allowDrop.bind(this);
+    }
+
+    init() {
+        this.inited = true;
+
+        // Label each selectable element with a class, and bind the click event
+        const providers = this.getPossibleProviders();
+        const providerSelectors = providers.map(p => p.selector);
+        $(providerSelectors.join(', '))
+            .addClass('ile-selectable')
+            .on('click', this.toggleSelected);
+
+        // Some providers need a "handle" to allow dragging.
```

Comment:
curious why this is

Reply:
It's to handle things that are only links; like this: 
![image](https://user-images.githubusercontent.com/6251786/168339729-740e7492-f67c-411d-be7c-cd0a8c113250.png)

(You can select two authors from here and merge them by clicking the "handle" (little dot))